llm_bioinformatics_research/src/References/References.css

```diff
@@ -0,0 +1,21 @@
+.references-container {
+    margin-top: 40px;
+  }
+  
+  .references-search {
+    margin-bottom: 20px;
+  }
+  
```

Comment:
It would make it easier to see the different references if they were a different color than the background.